lib/rouge/lexers/gettext_po.rb

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class GettextPO < RegexLexer
+      tag 'gettext_po'
+      aliases 'gettext', 'po'
+
+      title "Gettext Portable Object"
+      desc 'Gettext portable object files (gnu.org/software/gettext/'
+      filenames '*.po', '*.pot'
+      mimetypes 'text/x-gettext-translation'
+
+      def self.keywords
+        @keywords ||= Set.new %w(
+          msgctxt msgid msgid_plural msgstr
+        )
+      end
+
+      state :root do
+        rule %r/\s+/, Text::Whitespace
+
+        rule %r/^#[,:].*/, Comment::Special
+        rule %r/^#[^,:].*/, Comment
+
+        rule %r/".*?"/, Str::Double
```

Comment:
What if a string contains `"`? Do these need to be escaped? Should the rule be this:

```suggestion
        rule %r/"(?:\\\\|\\"|[^"])*"/, Str::Double
```

Reply:
@shoorick Just checking about the answer to this question.